clusterdock/models.py

```diff
@@ -475,3 +485,17 @@ def _add_node_to_etc_hosts(self):
                               command=[self.execute_shell, '-c', command],
                               volumes=volumes,
                               remove=True)
+
+
+def _get_clusterdock_label(cluster_name=None):
```

Comment:
Can we document this function a bit to describe what it's doing? Also, perhaps it would belong in the utils module instead of in the models one?